app/ui-react/syndesis/package.json

```diff
@@ -11,7 +11,6 @@
   },
   "dependencies": {
     "@allpro/react-router-pause": "^1.1.3",
-    "@craco/craco": "^5.0.2",
```

Comment:
It's a nitpick, but usually all the `@` packages are first.

Reply:
moved up in the fil